example/World.tsx

```diff
@@ -0,0 +1,31 @@
+import Vue from 'vue'
+import Component from '../lib/index'
+
+@Component({
+  props:{
+    title:{
+      type:String,
+      required:true
+    }
+  }
+})
+export default class World extends Vue {
+  title:string
+  content:string = 'default content'
+  get computedContent () {
+    return 'computed ' + this.content
+  }
+
+  clearContent() {
+    this.content = ''
+  }
+  render(h) {
+    return <div >
+      <h1>{this.title}</h1>
```

Comment:
IIRC tsx requires additional type declaration under `strict: true` flag. Will this compile>